### PR TITLE
[AKS] update idletimeout to 100 minutes

### PR DESCRIPTION
**Pull request by gtracer**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Comment by yungezz** [comment]
hi @arrownj could you pls review the PR? thanks

**Comment by arrownj** [comment]
Hi @gtracer , as talked before, the feature PR and release PR should be separated in case there are other features in one release.

File: src/aks-preview/HISTORY.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.4.65
++++++
```
Comment (arrownj) [inline src/aks-preview/HISTORY.md:6]: Please submit another PR to release new version.

**Comment by arrownj** [comment]
Hi @gtracer , is this PR still valid ?

**Review by robbiezhang — APPROVED** [review]

**Comment by gtracer** [comment]
> Hi @gtracer , is this PR still valid ?

Yes

**Review by JunSun17 — APPROVED** [review]
Looks good to me.
